content/02.main-text.md

```diff
@@ -211,6 +211,19 @@ manubot cite --render --format=markdown \
   pmid:29618526 doi:10.1038/550143a
 ```
 
+### Interactive features and appearance
+
+Manubot comes with several "plugins" that can be included in manuscripts exported as HTML.
+These plugins add special interactive features that enhance the user experience of viewing and reading manuscripts (@fig:plugins).
+The build process can also accommodate different "themes", which change the general aesthetics and appearance of the exported document (for example, from a contemporary sans-serif style to a more traditional serif style).
+
+![
+Screenshots of the various Manubot plugins, illustrating their functionality and usefulness.
```

Comment:
This is a bad sentence, but do we want to add something like:

> For example, citation tooltips provide users with access upon hovering to the reference and enable navigated to all other instances of that a given citation.

I think that commenting on at least one of the panels could help users approach this figure.

Reply:
How about this:

> For example, with the "tooltips" plugin included, when the user hovers over a link to a reference or figure, a preview of that item pops up over the link, along with controls to navigate between other mentions of that item elsewhere in the document.

I'd put it right after the first sentence.

Reply:
> For example, with the "tooltips" plugin enabled, when the user hovers over a link to a reference or figure, a preview of that item pops up over the link, along with controls to navigate between other mentions of that item elsewhere in the document.

This still needs to be added.

Also we should probably attribute the source of the screenshots:

```markdown
All screenshots show snippets from the[ Sci-Hub Coverage Study](https://greenelab.github.io/scihub-manuscript/v/fd7acb7ed0108c920da56f84819ce13f02f68aa8/) besides for the Hypothesis screenshot, which was taken from the [TPOT-DS manuscript](https://trang1618.github.io/tpot-ds-ms/).
Both manuscripts are available under a [CC BY 4.0 License](https://creativecommons.org/licenses/by/4.0/ "Creative Commons Attribution 4.0 International").
```

Reply:
I did add the example tooltip sentence. It's showing up for me in the diff.

I can put the screenshot attribution in the figure caption? Is it necessary to state the license in this manuscript, when they can follow the links to the other manuscripts if they need that info? Idk if it's important enough to list here?

Reply:
> I did add the example tooltip sentence. It's showing up for me in the diff.

Ah I see. I was envisioning the sentence would go in the caption, since I think it's most useful for users when they are directly viewing the figure. Not a strong preference however.

> Is it necessary to state the license in this manuscript

Yes, it is best practice. As per the non-legal (more readable) CC BY license terms:

> Attribution — You must give appropriate credit, **provide a link to the license**, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.

Bolding is mine.

We might also want to state in the caption that our markup is in purple, not just for the CC BY modifications clause compliance, but also so there is no uncertainty.

Reply:
I'm aware that you have to provide a link to the license and all that, but presumably that is already done in the source manuscripts... But if you say it is normal practice I'll put it in. Seems odd to duplicate info about the source manuscripts in every other manuscript that references it.

Reply:
Great! Can you bold the first sentence of the caption? In other contexts, this will become the figure caption title.